JustSaying/Messaging/MessageHandling/MessageContextAccessor.cs

```diff
@@ -0,0 +1,15 @@
+using System.Threading;
+
+namespace JustSaying.Messaging.MessageHandling
+{
+    public class MessageContextAccessor : IMessageContextReader, IMessageContextAccessor
```

Comment:
`internal sealed`?

Reply:
It needs to be public for DI to be able to register it. Otherwise we'll be going down the `InternalsVisibleTo` route.